docs/docs/technology/architecture.md

```diff
@@ -6,7 +6,9 @@ Composable’s architecture is thoughtfully constructed to actualize this missio
 
 These different components collectively facilitate a complete, end-to-end ecosystem for cross-chain DeFi. In brief, the core components of the technical architecture are as follows:
 
-## [The CVM](cvm.md)
+## [MANTIS](mantis.md)
+Multichain Agnostic Normalized Trust-minimized Intent Settlement (MANTIS) is an ecosystem-agnostic intent settlement framework. It is the culmination of all other elements of our architecture; it uses our trust minimized bridge (and thus the IBC as well as all three of our different chains) as well as the CVM to facilitate cross-chain execution of user intents. As a result, we are able to provide users with a novel, intuitive, and streamlined experience for participating in cross-chain DeFi. 
```

Comment:
it uses Composable's trust minimized bridges

Reply:
edited